doc/guides/keycloak_protection.md

```diff
@@ -1,68 +1,118 @@
 ## Prerequisites:
 
-- You have provisioned Keycloak as described in [AeroGear Docs](https://docs.aerogear.org/aerogear/latest/identity-management.html).
+- There is a Keycloak service available.
+- You must first create a client for your application in the Keycloak Administration Console, and then click on the `Installation` tab, select `Keycloak OIDC JSON` for `Format` Option, and then click on `Download`. Save the downloaded `keycloak.json` file to your project.
 
 ## Setting up Keycloak protection in sync framework
```

Comment:
Minor thing: can we fix the capitalisation and naming? Should probably be something like `Setting up Keycloak Protection in the Voyager Framework`